package.json

```diff
@@ -6,6 +6,7 @@
   "private": true,
   "scripts": {
     "bootstrap": "lerna bootstrap",
+    "bootstrap-example": "rm -rf example-project/node_modules/@logtail && ln -s ../../packages example-project/node_modules/@logtail",
```

Comment:
Interesting.  I think the bootstrap command works fine for me to link the libraries to be used in the example project.
For other repos I always use `yarn link`:
yarn link @logtail/{core,tools,types,node,browser,js,winston,pino,bunyan}

But if this works then no problem.

Reply:
`lerna bootstrap` only creates the links among the packages. Example project uses currently released version. `yarn link` can work as well, I usually created the symlink manually (to avoid having to think which package I need to use)